src/rade_api.h

```diff
@@ -71,6 +71,12 @@ extern "C" {
 #define RADE_MODEM_SAMPLE_RATE 8000           // modem waveform sample rate
 #define RADE_SPEECH_SAMPLE_RATE 16000         // speech sample rate
 
+// init rade_open() flags
+#define RADE_USE_C_ENCODER 0x1
```

Comment:
Question: is there a scenario where one would actually want to use Python for RX or TX (and C for the opposite)? If not, should these two be consolidated into one flag?

Reply:
For testing, e.g. C encoder driving Python decoder.  If you swapped both to C and a test failed you wouldn't know which one had bugs.